tkn/infra-aws-mac.yaml

```diff
@@ -99,7 +99,7 @@ spec:
 
   steps:
   - name: provisioner
-    image: quay.io/redhat-developer/mapt:v0.7.0-dev
+    image: <IMAGE>
```

Comment:
We need to take care of  

`app.kubernetes.io/version: "0.7.0-dev"` 

on the annotations part of the yaml.

This applies to all other yaml files

Reply:
sed command also covered the label now.

Reply:
:+1: